<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnicodeChecker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8,1">
        <item>
         <widget class="QLabel" name="label_char">
          <property name="text">
           <string>请输入字符串</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_String"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_see_check">
          <property name="text">
           <string>展示所有字符Unicode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,1">
        <item>
         <widget class="QLabel" name="label_string">
          <property name="text">
           <string>请输入单个字符</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Char"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_show_check_res">
          <property name="text">
           <string>字符判定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="plainTextEdit"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="chk_digit">
              <property name="text">
               <string>isDigit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chk_letter">
              <property name="text">
               <string>isLetter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_di_le">
              <property name="text">
               <string>isLetterOrNumber</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chk_upper">
              <property name="text">
               <string>isUpper</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chk_lower">
              <property name="text">
               <string>isLower</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chk_mark">
              <property name="text">
               <string>isMark</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_space">
              <property name="text">
               <string>isSpace</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_symbol">
              <property name="text">
               <string>isSymbol</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_punct">
              <property name="text">
               <string>isPunct</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>功能</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
